openc3-script-runner-api/config/routes.rb

```diff
@@ -15,7 +15,6 @@
 
     get  "/running-script" => "running_script#index"
     get  "/running-script/:id" => "running_script#show"
-    post "/running-script/:id/start" => "running_script#start"
```

Comment:
How is it just this route is removed?

Reply:
There is no running_script#start method.